project.clj

```diff
@@ -103,6 +105,8 @@
                "crux-metrics/test"
                "crux-s3/test"
                "crux-azure-blobs/test"
+               "crux-sql/crux-sql"
```

Comment:
suspect these want to be `crux-sql/test`?

Reply:
Fixed.